README.md

```diff
@@ -68,8 +68,8 @@ Retreat from the xupptumblers!
 ```javascript
 #!/usr/bin/env node
 require('yargs') // eslint-disable-line
-  .command('serve [port]', 'start the server', (yargs) => {
-    yargs
```

Comment:
What's the motivation for shortening this from `yargs` to `y`? Were you thinking because the object might be either a `yargs` or `yargsa` instance.

If we're going to have the asynchronous API surface in `yargs` itself, I think I'd prefer that we have something like:

```
const yargs = require('yargs').async
```

Similar to `require('fs').promises` in Node itself, at which point I'd just keep calling the API `yargs` in samples like this.